doc/nrf/libraries/networking/azure_iot_hub.rst

```diff
@@ -91,16 +91,16 @@ To connect to Azure IoT Hub without using DPS, complete the following minimum re
 1. In the `Azure Portal`_, navigate to :guilabel:`IoT Hub` and select the desired IoT hub.
 #. In the overview page, locate and copy the ``Hostname`` and configure :kconfig:option:`CONFIG_AZURE_IOT_HUB_HOSTNAME` to this address.
 
-   The hostname can also be set at run time.
-#. Set the option :kconfig:option:`CONFIG_AZURE_IOT_HUB_DEVICE_ID` to the device ID.
+   You can also set the hostname at run time.
```

Comment:
I'm pretty sure that "hostname" in running text should be "host name". It's "host name" once on the page, and "hostname" a whole bunch of times.

Reply:
Changed all of them.